litert/vendors/qualcomm/core/builders/prelu_op_builder.cc

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Qualcomm Innovation Center, Inc.
+// All Rights Reserved.
+
+#include <vector>
+
+#include "litert/vendors/qualcomm/core/builders/op_builder.h"
+#include "litert/vendors/qualcomm/core/tensor_pool.h"
+#include "litert/vendors/qualcomm/core/wrappers/op_wrapper.h"
+#include "litert/vendors/qualcomm/core/wrappers/tensor_wrapper.h"
+#include "QnnOpDef.h"  // from @qairt
+
+namespace qnn {
+
+std::vector<OpWrapper> BuildPreluOp(
+    TensorPool& tensor_pool, const std::vector<TensorWrapperRef>& inputs,
+    const std::vector<TensorWrapperRef>& outputs) {
+  std::vector<OpWrapper> res;
+
+  auto& prelu_op = CreateOpWrapper(res, QNN_OP_PRELU);
+  prelu_op.AddInputTensor(inputs[0]);
+  prelu_op.AddInputTensor(inputs[1]);
```

Comment:
I'm not sure if the number of inputs of prelu is guaranteed in LiteRt.
If it's not, I would prefer to switch to a for-loop.
That way, we can make sure the number of inputs for PReLu is always correct by QNN op validation process and avoid any issues. 
```suggestion
  for (const auto& input : inputs) {
    prelu_op.AddInputTensor(input);
  }
```